modules/commons/src/main/java/org/apache/synapse/commons/vfs/VFSUtils.java

```diff
@@ -373,6 +373,17 @@ public static boolean isFailRecord(FileSystemManager fsManager, FileObject fo) {
         return false;
     }
 
```

Comment:
shall we add a java doc comment ( format of the waitTimeBeforeRead s or ms ) 
and a check for waitTimeBeforeRead > 0